data_parser.py

```diff
@@ -771,8 +771,11 @@ class EntityInteract(Struct):
 
 class EntityMessage(Struct):
     """packet type: 51"""
-    unknown = Byte
-    target_id = SBInt32
+    target_unique = Flag
```

Comment:
Figured it out after messages from ancient gates were getting garbled. Turns out ancient gates have a unique entity ID.